aleph/model/bookmark.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime
+from normality import stringify
+from aleph.core import db
+from aleph.model.common import ENTITY_ID_LEN, IdModel
+
+
+class Bookmark(db.Model, IdModel):
```

Comment:
`aleph.model.common.DatedModel` defines `created_at` and `updated_at` timestamps and also includes a few utility methods. However, bookmarks don’t have an `updated_at` timestamp.